tests/v2/validation/provisioning/rke1/post_kdm_oob_release_test.go

```diff
@@ -0,0 +1,162 @@
+	require.NoError(k.T(), err)
+
+	providerName := k.providers[0]
+	provider := CreateProvider(providerName)
+
+	nodePools := []*management.NodePool{}
+	nodePoolNames := []string{}
+	clusterNames := []string{}
+	clusterResps := []*management.Cluster{}
+
+	for _, k8sVersion := range k.rke1kubernetesVersions {
+		for _, cni := range k.cnis {
+			logrus.Info("provisioning " + k8sVersion + " cluster..")
+			nodeTemplate, err := provider.NodeTemplateFunc(client)
+			require.NoError(k.T(), err)
+
+			clusterResp, nodePool, nodePoolName, clusterName, err := k.provisionRKE1Cluster(client, provider, k.nodesAndRoles, k8sVersion, cni, nodeTemplate)
+			require.NoError(k.T(), err)
+
+			nodePoolNames = append(nodePoolNames, nodePoolName)
+			clusterNames = append(clusterNames, clusterName)
+			clusterResps = append(clusterResps, clusterResp)
+			nodePools = append(nodePools, nodePool)
+		}
+	}
+
+	k.checkClustersReady(client, clusterResps, nodePools, clusterNames, nodePoolNames)
+
+}
+
+func (k *KdmChecksTestSuite) provisionRKE1Cluster(client *rancher.Client, provider Provider, nodesAndRoles []nodepools.NodeRoles, k8sVersion, cni string, nodeTemplate *nodetemplates.NodeTemplate) (*management.Cluster, *management.NodePool, string, string, error) {
+	clusterName := namegen.AppendRandomString(provider.Name)
+
+	cluster := clusters.NewRKE1ClusterConfig(clusterName, cni, k8sVersion, client)
+	clusterResp, err := clusters.CreateRKE1Cluster(client, cluster)
+	require.NoError(k.T(), err)
+
+	nodePool, err := nodepools.NodePoolSetup(client, nodesAndRoles, clusterResp.ID, nodeTemplate.ID)
+	require.NoError(k.T(), err)
+
+	nodePoolName := nodePool.Name
+
+	return clusterResp, nodePool, nodePoolName, clusterName, nil
+}
+
+func (k *KdmChecksTestSuite) checkClustersReady(client *rancher.Client, clusterResps []*management.Cluster, nodePools []*management.NodePool, clusterNames []string, nodePoolNames []string) {
+	for i, clusterResp := range clusterResps {
+		opts := metav1.ListOptions{
+			FieldSelector:  "metadata.name=" + clusterResp.ID,
+			TimeoutSeconds: &defaults.WatchTimeoutSeconds,
+		}
+
+		logrus.Info("waiting for cluster ", clusterResp.Name, " with k8s version ", k.rke1kubernetesVersions[i], " to be up..")
+		watchInterface, err := k.client.GetManagementWatchInterface(management.ClusterType, opts)
+		require.NoError(k.T(), err)
+
+		checkFunc := clusters.IsHostedProvisioningClusterReady
+
+		err = wait.WatchWait(watchInterface, checkFunc)
+		require.NoError(k.T(), err)
+		assert.Equal(k.T(), clusterNames[i], clusterResp.Name)
+		assert.Equal(k.T(), nodePoolNames[i], nodePools[i].Name)
+		assert.Equal(k.T(), k.rke1kubernetesVersions[i], clusterResp.RancherKubernetesEngineConfig.Version)
+
+		logrus.Infof("waiting for all pods to be up..")
+		podResults, podErrors := pods.StatusPods(client, clusterResp.ID)
+		assert.NotEmpty(k.T(), podResults)
+		assert.Empty(k.T(), podErrors)
+
+		logrus.Info(clusterResp.Name, " provisioned successfully")
```

Comment:
Same as above; logging can be removed as the `IsHostedProvisioningClusterReady` already provides logging when the cluster is `Active`.

Reply:
removed it.